superset/utils/webdriver.py

```diff
@@ -200,7 +198,7 @@ def get_screenshot(
             ]
             logger.debug("Wait %i seconds for chart animation", selenium_animation_wait)
             sleep(selenium_animation_wait)
-            logger.info(
+            logger.debug(
```

Comment:
@craig-rueda I left a question for you about this on my PR, to see if it's safe to keep as an info: https://github.com/apache/superset/pull/23114/files#diff-ee5af2b5ab4555c2f1d3d31d44fecd1c14bf03e13aa88cb0d15862d7dcea57bbL220